pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/tissue/TissueSMIDDao.java

```diff
@@ -297,4 +300,28 @@ public static List<String> getSmIdPksForTissue(String tissueId) {
         logger.info("Got %d smId pks in DSM DB for tissue with id %s", smIds.size() , tissueId);
         return smIds;
     }
+
+    public SmId get(int id) {
```

Comment:
How about a method name like `findById()`?  `get()` implies a simple return of state, whereas this method is doing a db lookup.